### ISSUE TITLE
Forms with Checkboxes or Radio button fields in succession break

### ISSUE THREAD

**Issue by jnicotra** [elementor/elementor#7660]
## Prerequisites

- [ x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x ] The issue still exists against the latest stable version of Elementor.

## Description

I have noticed an issue with the latest version of Elementor forms where a form that has either radio buttons or checkbox fields right after each other, the data seems to get messed up and the forms do not display properly.

See it here: https://imgur.com/a/GtC8syD

How it should display:  https://imgur.com/a/SHdieZ6

I thought it might be a plugin conflict, however, I replicated this issue on a separate development website also.

## Steps to reproduce

Create a form with a successive radio buttons or checkboxes.

## Isolating the problem

- [ x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x ] This bug happens with a default WordPress theme active.
- [ x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
I copied this AFTER re-enabling  my plugins and theme:
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.18
	MySQL version: (Ubuntu) v5.7.25
	PHP Version: 7.0.33-0ubuntu0.16.04.3
	PHP Max Input Vars: 1000
	PHP Max Post Size: 1000M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.0.4
	Site URL: http://bluepointdigital.com/sites/playground
	Home URL: http://bluepointdigital.com/sites/playground
	WP Multisite: No
	Max Upload Size: 1,000 MB
	Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: OceanWP
	Version: 1.6.4
	Author: Nick
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.4
		Author: WordPress Contributors

	Elementor
		Version: 2.5.10
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.2
		Author: Elementor.com

	Ocean Elementor Widgets
		Version: 1.1.4
		Author: OceanWP

	Ocean Extra
		Version: 1.5.3
		Author: OceanWP

	Ocean Sticky Header
		Version: 1.1.11
		Author: OceanWP

	Post Snippets
		Version: 3.0.8
		Author: GreenTreeLabs

	Short Code Elementor
		Version: 1.0
		Author: BPD Team


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	SSO
		Version: 0.1
		Author: Garth Mortensen, Mike Hansen


== Log ==
	: 
JS: showing 5 of 5JS: 2019-03-12 17:54:40 [error X 2][http://bluepointdigital.com/sites/playground/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.1:2:238528] Cannot read property \'statusText\' of undefined 
JS: 2019-03-12 19:07:34 [error X 6][http://bluepointdigital.com/sites/playground/wp-content/plugins/elementor/assets/js/editor-modules.js?ver=2.5.5:736:35] self.attributes[control.name].each is not a function 
JS: 2019-03-12 19:08:18 [error X 1][http://bluepointdigital.com/sites/playground/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=2.4.6:2:13911] t.get(...).get(...).where is not a function 
JS: 2019-03-12 19:11:34 [error X 1][http://bluepointdigital.com/sites/playground/wp-admin/load-scripts.php?c=0&load%5B%5D=underscore,jquery-core,jquery-migrate,backbone,shortcode,utils,wp-util,wp-backbone,media-models,moxiejs,plupload,wp-plupload,jqu&load%5B%5D=ery-ui-core,jquery-ui-widget,jquery-ui-mouse,jquery-ui-sortable&ver=5.0.3:16:5345] Cannot read property \'button_text\' of undefined 
JS: 2019-03-15 02:37:22 [error X 2][http://bluepointdigital.com/sites/playground/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.5.5:2:53459] Cannot read property \'noUiSlider\' of undefined 

PHP: showing 10 of 10PHP: 2019-03-12 19:02:33 [notice X 7][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::202] Undefined index: post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 12:14:42 [notice X 16][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::31] Array to string conversion [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 12:15:51 [notice X 11][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::28] Undefined offset: 0 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 12:30:39 [notice X 2][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::23] Undefined property: BPDElementor\Bpd_Elementor_Short_Code::$namekey [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 12:31:12 [notice X 10][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::18] Undefined variable: itfargs [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 12:33:04 [notice X 12][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::24] Undefined index: id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 13:29:07 [error X 1][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::88] Uncaught Error: Class 'BPDElementor\Controls_Manager' not found in /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php:88
Stack trace:
#0 /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor/includes/base/controls-stack.php(1919): BPDElementor\Bpd_Elementor_Short_Code->_register_controls()
#1 /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor/includes/base/controls-stack.php(485): Elementor\Controls_Stack->init_controls()
#2 /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor/includes/base/widget-base.php(160): Elementor\Controls_Stack->get_stack()
#3 /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor/includes/managers/widgets.php(274): Elementor\Widget_Base->get_stack(false)
#4 [internal function]: Elementor\Widgets_Manager->ajax_get_widget_types_controls_config(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#5 /home/bpdigital/public_html/sites/playgro [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 13:29:16 [error X 1][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::13] Class 'BPDElementor\Widget_Base' not found [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-20 13:29:34 [error X 1][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/class/bpd_elementor_code.php::83] Uncaught Error: Class 'BPDElementor\Bpd_Elementor_Short_Code' not found in /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/class/bpd_elementor_code.php:83
Stack trace:
#0 /home/bpdigital/public_html/sites/playground/wp-includes/class-wp-hook.php(286): ElementorCustomCodeElement->widgets_registered('')
#1 /home/bpdigital/public_html/sites/playground/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#2 /home/bpdigital/public_html/sites/playground/wp-includes/plugin.php(453): WP_Hook->do_action(Array)
#3 /home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor/includes/plugin.php(475): do_action('elementor/init')
#4 /home/bpdigital/public_html/sites/playground/wp-includes/class-wp-hook.php(286): Elementor\Plugin->init('')
#5 /home/bpdigital/public_html/sites/playground/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array)
#6 /home/bpdigital/public_html/sites/playground/wp-includes/plugin.php(453): WP_Hook->do_action(Array) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2019-03-25 12:34:03 [notice X 1][/home/bpdigital/public_html/sites/playground/wp-content/plugins/elementor-shotcut/elementor/bpd_short_code.php::35] Use of undefined constant lot_id - assumed 'lot_id' [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 282019-03-29 05:03:56 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Start  
2019-03-29 05:03:58 [info] Elementor Pro/Upgrades - _v_2_5_0_posts callback needs to run more 2 times 
2019-03-29 05:03:58 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Start  
2019-03-29 05:04:00 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Finished 
2019-03-29 05:04:00 [info] Elementor Pro/Upgrades - _v_2_5_0_products Start  
2019-03-29 05:04:02 [info] Elementor Pro/Upgrades - _v_2_5_0_products Finished 
2019-03-29 05:04:02 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Start  
2019-03-29 05:04:04 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Finished 
2019-03-29 05:04:04 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Start  
2019-03-29 05:04:04 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Finished 
2019-03-29 05:04:05 [info] Elementor Pro/Upgrades - _v_2_5_0_form Start (2 of 5, 40%) 
2019-03-29 05:04:05 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Start (2 of 3, 67%) 
2019-03-29 05:04:06 [info] Elementor Pro/Upgrades - _v_2_5_0_form Start (3 of 5, 60%) 
2019-03-29 05:04:10 [info] elementor-pro::elementor_pro_updater Started 
2019-03-29 05:04:10 [info] Elementor Pro/Upgrades - _v_2_5_0_form Start (4 of 5, 80%) 
2019-03-29 05:04:15 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Start (3 of 3, 100%) 
2019-03-29 05:04:17 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Finished 
2019-03-29 05:04:17 [info] Elementor Pro/Upgrades - _v_2_5_0_form Start (5 of 5, 100%) 
2019-03-29 05:04:19 [info] Elementor Pro/Upgrades - _v_2_5_0_form Finished 
2019-03-29 05:04:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.6',
  'to' => '2.5.2',
)]


				
```
</details>

**Comment by prhelms86** [comment]
Similar issue here. When a user attempts to complete a form that has radio buttons, an error is received. I have attempted to create new forms and change settings but no luck.

**Comment by shilo-ey** [comment]
Hi @jnicotra @prhelms86 

This issue most probably has been fixed in Elementor Pro v2.5.3

Feel free to update!

**Comment by jnicotra** [comment]
Updated and the issue persists.

**Comment by shilo-ey** [comment]
Hi @jnicotra 

Please follow #7711 for updates.


Thanks!

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.5.3

Feel free to update!
